web/pdf_find_controller.js

```diff
@@ -322,10 +322,9 @@ var PDFFindController = (function PDFFindControllerClosure() {
      * @param {number} index - match index.
      * @param {Array} elements - text layer div elements array.
      * @param {number} beginIdx - start index of the div array for the match.
-     * @param {number} endIdx - end index of the div array for the match.
```

Comment:
@Snuffleupagus Do you agree with the above or do you want me to keep this variable?